spec/controllers/my_controller_spec.rb

```diff
@@ -98,8 +98,8 @@
                                new_password_confirmation: 'adminADMIN!New'
       end
 
-      it 'should redirect to the my account page' do
-        expect(response).to redirect_to('/my/account')
+      it 'should redirect to the my password page' do
+        expect(response).to redirect_to('/my/password')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.